api/queries/channel/metaData.js

```diff
@@ -28,7 +27,13 @@ export default async (root: DBChannel, _: any, ctx: GraphQLContext) => {
 
   // Cache the fields for an hour
   (await typeof cachedOnlineMemberCount) === 'number' ||
-    cache.set(`channel:${id}:onlineMemberCount`, onlineMemberCount, 'ex', 3600);
+    cache.set(
+      channelOnlineMemberCount(id),
+      onlineMemberCount,
+      'NX',
```

Comment:
I don't think we want NX here, why did you add that? I don't know what that does.

Reply:
`NX` is part of the redis signature now. `NX` says: if the record for this key doesn't exist, create it. `EX` following says 'set the expieration for this key'

Reply:
https://stackoverflow.com/a/16296998

Reply:
Actually I think that comment is misleading...